conboarding.md

```diff
@@ -7,7 +7,11 @@ Please note that some of the following links and resources are not publicly acce
 
 ### GitHub
 
-Ask Yarik to add you to the [CON Org](https://github.com/con).
+1. Ask Yarik to add you to the [CON Org](https://github.com/con), including possibly the 'centroids' team.
+
+1. Add yourself to [whoweare](https://github.com/con/centerforopenneuroscience.org/blob/master/content/pages/whoweare.html) to be displayed on [the website](https://centerforopenneuroscience.org/whoweare).
```

Comment:
I found that doing this step earlier rather than later enabled other things to happen quicker and easier